script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_1_1.txt

```diff
@@ -5,20 +5,20 @@ At any rate, I decide to greet Akiha first.
 "...Good morning, Nii-san.
 That was a rather lackluster greeting, did you not sleep well last night? You look like you are on the verge of collapse."
 <Page3>
-"Unfortunately, that's just my face. Don't expect me to greet you with high spirits in the morning."
+"Unfortunately, that's just my face. Don't expect me to greet you with such high spirits in the morning."
 Regardless of what happened last night, I don't want my greetings to make people think that I'm some kind of delinquent, staying up late.
 <Page4>
 "Good morning, Shiki-san.
 If you would like some breakfast, I can serve it in the dining room, so please feel free to ask me anytime."
-On the other hand, Kohaku-san has this soothing effect. Like the north wind and the sun maybe?
+On the other hand, Kohaku-san has this soothing effect. They're like the north wind and the sun maybe?
 <Page5>
 "Thank you very much.
-But have you finished already, Akiha? From the looks of it, you're having a post-breakfast tea."
+But have you finished already, Akiha? From the looks of it, you're having post-breakfast tea."
 <Page6>
 "Of course. I do not care what time you wake up, but at least make sure you have enough time for breakfast.
 Having breakfast after seven o'clock is a sign of sluggishness."
 <Page7>
-I resist the urge to say, 'No, seven o'clock is normal. This house is not normal to begin with'.
+I resist the urge to say, 'No, seven o'clock is normal'. This house is the one that's abnormal.
```

Comment:
Suggest 'It's this household that's abnormal.' based on current edits, otherwise it implies it's the mansion that's abnormal rather than Akiha.